universe/config.json

```diff
@@ -72,9 +72,9 @@
                     "default": false
                 },
                 "agent-port": {
-                    "description": "The port that Jenkins agents will use to connect to the master. The default port is set to 50000; a value of 0 will result in a random port being chosen.",
+                    "description": "The port that Jenkins agents will use to connect to the master. The default port is set to 30000; a value of 0 will result in a random port being chosen.",
```

Comment:
Maybe s/a random port/an arbitrary available port/ ? 
After all, the choice is not truly random, and nothing makes this choice before the set of ports available on the agent becomes known.

Reply:
Sounds reasonable, applying this.